dim/src/scanners/scanner_daemon.rs

```diff
@@ -95,7 +90,7 @@ impl FsWatcher {
             let matcher = super::get_matcher(&self.tx);
 
             if let Ok(mfile) = extractor
-                .mount_file(path.clone(), self.library_id, self.media_type)
+                .mount_file(path.clone(), self.library_id, self.media_type, false)
```

Comment:
I think if we set this to `true` then whenever we receive a `Write` event we would update the metadata that we have on the file, so the issue where we have wrong or unknown video metadata will go away.